tidy3d/components/spice/analysis/dc.py

```diff
@@ -0,0 +1,74 @@
+"""
+This class defines standard SPICE electrical_analysis types (electrical simulations configurations).
+"""
+
+from typing import Optional, Tuple, Union
+
+import pydantic.v1 as pd
+
+from tidy3d.components.base import Tidy3dBaseModel
+from tidy3d.components.spice.sources.types import VoltageSourceType
+from tidy3d.components.tcad.types import HeatChargeMonitorTypes
+from tidy3d.components.types import annotate_type
+
+
+class ChargeToleranceSpec(Tidy3dBaseModel):
+    """
+    This class sets some Charge tolerance parameters.
+
+    Example
+    -------
+    >>> import tidy3d as td
+    >>> charge_settings = td.ChargeToleranceSpec(abs_tol=1e8, rel_tol=1e-10, max_iters=30)"""
+
+    abs_tol: pd.PositiveFloat = pd.Field(
+        default=1e10,
+        title="Absolute tolerance.",
+        description="Absolute tolerance used as stop criteria when converging towards a solution. Should be "
+        "equivalent to the"
+        "SPICE ABSTOL DC transfer parameter. TODO MARC units, TODO check equivalence. TODO what does this mean?",
+    )
+
+    rel_tol: pd.PositiveFloat = pd.Field(
+        default=1e-10,
+        title="Relative tolerance.",
+        description="Relative tolerance used as stop criteria when converging towards a solution.  Should be equivalent to the"
+        "SPICE RELTOL DC transfer parameter. TODO MARC units, TODO check equivalence. TODO what does this "
+        "mean?",
+    )
+
+    max_iters: pd.PositiveInt = pd.Field(
```

Comment:
@daquinteroflex I have reverted these names to the short version. Basically, my new approach to this is: if the spellchecker doesn't complain and it's common enough I'll take the shortened version, otherwise, full names